### ISSUE TITLE
Log sigmoid

### ISSUE THREAD

**Issue by vladfi1** [tensorflow/tensorflow#3719]
It would be nice to have a numerically stable log_sigmoid similar to the existing log_softmax.

**Comment by girving** [comment]
Do you mean `-tf.softplus`?

**Comment by girving** [comment]
I changed my mind, after accidentally writing `log_sigmoid(x) = -tf.nn.softplus(x)` instead of the correct `-tf.nn.softplus(-x)`.  We should have `tf.log_sigmoid`, though it should be implemented as pure Python.
